runtime/syntax/python.vim

```diff
@@ -113,7 +112,7 @@ syn match   pythonEscape	"\\\o\{1,3}" contained
 syn match   pythonEscape	"\\x\x\{2}" contained
 syn match   pythonEscape	"\%(\\u\x\{4}\|\\U\x\{8}\)" contained
```

Comment:
This is missing some changes on lines 97, 100, 103, 106 from the original patch. I'll fix it when merging.